### PR TITLE
Adds very common (3+) crayon prefabs for maps

### PR DESCRIPTION
**Pull request by Bilbo367**
## About The Pull Request

Adds a bunch of crayon prefabs for the maps.

## Why It's Good For The Game

Reduces 45 map Var edits, and completely removes some Var edits.

### CODE REVIEW

File: _maps/RandomZLevels/museum.dmm [hunk 1]
```diff
@@ -518,17 +518,9 @@
...
-	},
-/obj/effect/decal/cleanable/crayon{
-	icon_state = "i"
-	},
-/obj/effect/decal/cleanable/crayon{
-	icon_state = "e";
-	pixel_x = 12
-	},
+/obj/effect/decal/cleanable/crayon/lie1,
+/obj/effect/decal/cleanable/crayon/lie2,
```
Comment (mc-oofert) [inline _maps/RandomZLevels/museum.dmm:522]: the pixel offsets are gone

Comment (mc-oofert) [reply _maps/RandomZLevels/museum.dmm:522]: nvm theyre on the object by default but thats
pretty stupid

Comment (MrMelbert) [reply _maps/RandomZLevels/museum.dmm:522]: There's nothing wrong with that, pixel offsets are not the devil

**Review by MrMelbert — CHANGES_REQUESTED** [review]
There's something to be said about making very common graffiti that are used across maps into a subtype (like runes) but for ones used on one map, like pre-made letters, all it does is clutter the map editor

File: _maps/RandomZLevels/museum.dmm [hunk 2]
```diff
@@ -523,7 +523,7 @@
 	pixel_x = -10
 	},
 /obj/effect/decal/cleanable/crayon{
-	icon_state = "i"
+	icon_state = "l"
```
Comment (Ghommie) [inline _maps/RandomZLevels/museum.dmm:526]: Why are you changing `i` to `l`?

File: _maps/RandomZLevels/museum.dmm [hunk 3]
```diff
@@ -523,7 +523,7 @@
...
 	},
 /obj/effect/decal/cleanable/crayon{
 	icon_state = "e";
```
Comment (Ghommie) [inline _maps/RandomZLevels/museum.dmm:529]: These could use have own subtype too.

File: code/game/objects/effects/decals/crayon.dm [hunk 4]
```diff
@@ -41,3 +41,29 @@
 		cost *= 0.5
 	spraycan.use_charges(user, cost, requires_full = FALSE)
 	return DONT_USE_SPRAYCAN_CHARGES
+
+///Common crayon decals in map.
+/obj/effect/decal/cleanable/crayon/rune4
+	icon_state = "rune4";
```
Comment (JohnFulpWillard) [inline code/game/objects/effects/decals/crayon.dm:47]: You need to remove all these ``;``'s

File: code/game/objects/effects/decals/crayon.dm [hunk 5]
```diff
@@ -41,3 +41,29 @@
...
+	paint_colour = "#DC143C"
```
Comment (JohnFulpWillard) [inline code/game/objects/effects/decals/crayon.dm:48]: Use paint color defines please

File: code/game/objects/effects/decals/crayon.dm (hunk 4 above)
Comment (Bilbo367) [reply code/game/objects/effects/decals/crayon.dm:47]: what do they even do?

Comment (JohnFulpWillard) [reply code/game/objects/effects/decals/crayon.dm]: Nothing good in code, it's only here cause you copied and pasted them from the maps.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**